### ISSUE TITLE
The "In Depth Guide" is down

### ISSUE THREAD

**Issue by ghost** [glamdevhero/node-puppeteer-instagram-bot#4]
It looks like glamdevhero.com is down and I can't access the guide you linked to. I got started going through it already and would love to continue if there's a way to access it again. Thanks for putting it together.

**Comment by CharlesConner** [comment]
Yeah great work making this, would wrote on how to build this came back online

**Comment by ghost** [comment]
I’m still unable to access it.
